test/test_prototype_transforms.py

```diff
@@ -1206,9 +1206,9 @@ def test__transform(self, mocker):
         bboxes = make_bounding_box(format="XYXY", image_size=(32, 24), extra_dims=(6,))
         label = features.Label(torch.randint(0, 10, size=(6,)))
         ohe_label = features.OneHotLabel(torch.zeros(6, 10).scatter_(1, label.unsqueeze(1), 1))
-        masks = make_segmentation_mask((32, 24))
-        ohe_masks = features.SegmentationMask(torch.randint(0, 2, size=(6, 32, 24)))
-        sample = [image, bboxes, label, ohe_label, masks, ohe_masks]
+        masks = make_segmentation_mask((32, 24), num_objects=6)
```

Comment:
We currently only work with object detection masks. They were called `ohe_masks` here before. I've renamed them and removed the tests for what was `masks` before.